sql-cli/sql_cli/__main__.py

```diff
@@ -113,9 +107,14 @@ def run(
         exists=True,
         help="path to variables yaml or json file",
     ),
```

Comment:
I believe we shouldn't expose this in 0.1 - it doesn't seem to fit our 0.1 user journey.